matchers/be_in_matcher_test.go

```diff
@@ -0,0 +1,56 @@
+package matchers_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/matchers"
+)
+
+var _ = Describe("BeIn", func() {
+	Context("when passed a supported type", func() {
+		It("should do the right thing", func() {
+			Expect(2).Should(BeIn([2]int{1, 2}))
+			Expect(3).ShouldNot(BeIn([2]int{1, 2}))
+
+			Expect(2).Should(BeIn([]int{1, 2}))
+			Expect(3).ShouldNot(BeIn([]int{1, 2}))
+
+			Expect(2).Should(BeIn(1, 2))
+			Expect(3).ShouldNot(BeIn(1, 2))
+
+			Expect("abc").Should(BeIn("abc"))
```

Comment:
This is a slightly confusing test and I wonder whether it also points to this really meaning `BeElementIn`. I could imagine trying to write:

`Expect("abc").To(BeIn("123abc456"))`

Reply:
Yeah, I see your point, that might be confusing indeed. Changed.